.github/CODEOWNERS

```diff
@@ -13,14 +13,14 @@
 /website/blog/                       @runleonarun
 
 
-
 # Adapter & Package Development Docs
-/website/docs/docs/supported-data-platforms.md/                       @runleonarun @dataders
-/website/docs/reference/warehouse-profiles/                     @runleonarun @dataders
-/website/docs/reference/resource-configs/                       @runleonarun @dataders
+/website/docs/docs/supported-data-platforms.md/                 @runleonarun @dataders
+/website/docs/reference/warehouse-setups/                       @runleonarun @dataders
+# `resource-configs` contains more than just warehouse setups
+/website/docs/reference/resource-configs/*-configs.md           @runleonarun @dataders
```

Comment:
@mirnawong1 perhaps a better answer would to be bifurcate the `resource-configs/` directory to make warehouse configs in their own folder? idk...